src/app/containers/app/app.component.ts

```diff
@@ -299,6 +287,7 @@ export class AppComponent implements OnInit, OnDestroy {
    * @param totalIdleTime - Should reach here in minutes
    */
   public initTimeoutNotificationService(idleModalDisplayTime, totalIdleTime) {
+    if (this.timeoutNotificationServiceInitialised) return;
```

Comment:
is this common to do shorthand if statements? Personally not a fan. But leave it if its common practice